common/components/BalanceSidebar/TokenBalances.jsx

```diff
@@ -0,0 +1,78 @@
+// @flow
+import React from 'react';
+import translate from 'translations';
+import TokenRow from './TokenRow';
+import AddCustomTokenForm from './AddCustomTokenForm';
+import type { TokenBalance } from 'selectors/wallet';
+import type { Token } from 'config/data';
+
+export default class TokenBalances extends React.Component {
+  props: {
+    tokens: TokenBalance[],
+    onAddCustomToken: (token: Token) => any,
+    onRemoveCustomToken: (symbol: string) => any
+  };
```

Comment:
I notice sometimes you define props like this, and sometimes you define props like you did in Line 18 of BalanceSidebar.jsx

https://github.com/MyEtherWallet/MyEtherWallet/pull/41/files#diff-1bf2753416d4d93e210e97f6912fec52R18

I personally don't mind either or, but I think having them be consistent would be nice :)